src/swish/web-mats/static/ok.html

```diff
@@ -1 +1 @@
-<html><body><p>ok</p></body></html>
+<html><body><p>ok</p></body></html>

```

Comment:
Why did we need to change the line endings of `ok.html` and `ok.txt`?

Reply:
The Travis-CI Windows machine's git implementation flips the line endings of these files to CRLF. I thought this would be the easiest way to deal with it.

Reply:
Travis-CI sets git config --global core.autocrlf to true, which is a bug that's been reported to Travis-CI.

Reply:
As a result, we could tweak the build script to turn this option off and reset the affected files. That's what I'm doing for Chez Scheme.